e2e/cypress/integration/integrations/builtin_commands/invite_people_command_spec.js

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @integrations
+
+import {loginAndVisitChannel} from './helper';
+
+describe('Integrations', () => {
+    let testUser;
+    let testTeam;
+    const userGroup = [];
+    let testChannelUrl;
+
+    before(() => {
+        cy.apiInitSetup().then(({team, user}) => {
+            testUser = user;
+            testTeam = team;
+
+            Cypress._.times(2, () => {
+                cy.apiCreateUser().then(({user: otherUser}) => {
+                    userGroup.push(otherUser);
+                });
+            });
+        });
+    });
+
+    beforeEach(() => {
+        cy.apiAdminLogin();
+        cy.apiCreateChannel(testTeam.id, 'channel', 'channel').then(({channel}) => {
+            testChannelUrl = `/${testTeam.name}/channels/${channel.name}`;
+            cy.apiAddUserToChannel(channel.id, testUser.id);
+        });
```

Comment:
No need to do this here at `beforeEach` since there's only one `it` block. Please move into the `before` hook.